bpdm-pool/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/repository/LegalEntityRepository.kt

```diff
@@ -37,6 +38,28 @@ interface LegalEntityRepository : PagingAndSortingRepository<LegalEntity, Long>,
 
     fun findByUpdatedAtAfter(updatedAt: Instant, pageable: Pageable): Page<LegalEntity>
 
+    /*@Query("SELECT DISTINCT p FROM LegalEntity p WHERE p.legalName.value = :value")
+    fun findByLegalNameValue(value:String, pageable: Pageable): Page<LegalEntity>*/
+
+    /*@Query("SELECT DISTINCT p, levenshtein_distance(p.legalName.value, :value) AS distance\n" +
+            "FROM LegalEntity p\n" +
+            "WHERE p.legalName.value LIKE :value\n" +
+            "ORDER BY distance DESC\n")
+    fun findByLegalNameValue(value: String, pageable: Pageable): Page<LegalEntity>*/
+
+    /*@Query(value = "SELECT p.*, SIMILARITY(p.legalName.value, :value) AS score " +
+            "FROM LegalEntity p " +
+            "WHERE SIMILARITY(p.legalName.value, :value) > 0.3 " + // Adjust the similarity threshold as needed
+            "ORDER BY score DESC",
+        countQuery = "SELECT COUNT(*) FROM LegalEntity p WHERE SIMILARITY(p.legalName.value, :value) > 0.3", // Count query
+        nativeQuery = true)
+    fun findByLegalNameValue(value:String, pageable: Pageable): Page<LegalEntity>*/
+
+    /*fun findByLegalNameValue(value: String, pageable: Pageable): Page<LegalEntity>*/
```

Comment:
Please remove these comments as they don't explain the function logic

Reply:
Yeah, removed it was there by mistake.